lib/stm32/f4/rcc.c

```diff
@@ -75,6 +75,7 @@ const struct rcc_clock_scale rcc_hse_8mhz_3v3[RCC_CLOCK_3V3_END] = {
 		.hpre = RCC_CFGR_HPRE_DIV_NONE,
 		.ppre1 = RCC_CFGR_PPRE_DIV_2,
 		.ppre2 = RCC_CFGR_PPRE_DIV_NONE,
+		.power_save = PWR_SCALE1,
```

Comment:
I think we should stop calling this power_save while we're here, it's called ```voltage_scale``` on l1/l0, and ```vos_scale``` everywhere else.  (We should even move l1/l0 to vos_scale as well, but that's out of scope of this)